src/sdl2.re

```diff
@@ -347,6 +347,7 @@ module Event = {
     isFling: bool,
     isInterrupt: bool,
     source: WheelType.t,
+    timestamp: int,
```

Comment:
I'm not sure I do now that I think of it. I think the sdl type is only a C int rather than a long. OCaml int type defaults to word length of platform correct?

Reply:
nvm, type is good (first coerces to right size using Val_int macro)